### DISCUSSION TITLE
Marker link

### DISCUSSION THREAD

**Discussion by jan-b** [mistic100/Photo-Sphere-Viewer#1434] [General]
Anyone that can tell me if it's possible to use a Marker as link to another page?
Without Virtual Tour plugin

**Comment by mistic100** [comment]
Next time please create a "Question" issue.

You use the select-marker event https://photo-sphere-viewer.js.org/plugins/markers.html#select-marker-marker-doubleclick-rightclick

**Reply by jan-b** [reply]
Of course,  will use "Question" next time.
It was not clear what the code will be, but will dig into your answer 😊

**Reply by jan-b** [reply]
Works perfect 😊
Thanks!
